frontend/seo/product-list/hreflangs.ts

```diff
@@ -0,0 +1,260 @@
+   'ipad-air-2': 'Parts/iPad_Air_2',
+   'ipad-6': 'Parts/iPad_6',
+   'iphone-6-plus': 'Parts/iPhone_6_Plus',
+   'ipad-air-1': 'Parts/iPad_Air',
+   'ipad-5': 'Parts/iPad_5',
+   'iphone-4s': 'Parts/iPhone_4S',
+   'ipad-2': 'Parts/iPad_2',
+   'iphone-5': 'Parts/iPhone_5',
+   'ipad-mini-retina': 'Parts/iPad_Mini_2',
+   'imac-upgrade-kits': 'Parts/iMac/Kits',
+   'apple-watch': 'Parts/Apple_Watch',
+   'iphone-5c': 'Parts/iPhone_5c',
+   macbook: 'Parts/MacBook',
+   'apple-macbook-parts': 'Parts/Mac_Laptop',
+   parts: 'Parts',
+};
+
+const euHreflangPaths: Record<Collection, string> = {
+   toolkits: 'toolkits',
+   'apple-iphone-parts': 'apple-iphone-parts',
+   'steam-deck-parts': 'steam-deck-parts',
+   'samsung-galaxy-parts': 'samsung-galaxy-parts',
+   'macbook-pro': 'macbook-pro',
+   'nintendo-switch-parts': 'nintendo-switch',
+   'android-parts': 'android-parts',
+   'macbook-air': 'macbook-air',
+   'mac-parts': 'mac-parts',
+   'iphone-11': 'iphone-11',
+   'google-phone': 'google-pixel-parts',
+   'apple-ipad-parts': 'apple-ipad-parts',
+   'iphone-12-parts': 'iphone-12-parts',
+   'iphone-12-pro-max-parts': 'iphone-12-pro-max-parts',
+   'iphone-xr': 'iphone-xr',
+   'game-console-parts': 'game-console-parts',
+   'iphone-x': 'iphone-x',
+   'google-pixel-parts': 'google-pixel-parts',
+   'iphone-12-mini-parts': 'iphone-12-mini-parts',
+   'iphone-12-pro-parts': 'iphone-12-pro-parts',
+   'oneplus-parts': 'oneplus-parts',
+   'iphone-xs': 'iphone-xs',
+   'iphone-8': 'iphone-8',
+   'iphone-xs-max': 'iphone-xs-max',
+   'iphone-7': 'iphone-7',
+   'iphone-11-pro-max': 'iphone-11-pro-max',
+   'iphone-7-plus': 'iphone-7-plus',
+   'apple-imac-parts': 'apple-imac-parts',
+   'samsung-galaxy-s9': 'samsung-galaxy-s9',
+   'valve-index-parts': 'valve-index-parts',
+   'samsung-galaxy-s10-parts': 'samsung-galaxy-s10-parts',
+   'samsung-galaxy-s10-plus': 'samsung-galaxy-s10-plus',
+   'iphone-se': 'iphone-se',
+   'iphone-11-pro': 'iphone-11-pro',
+   'iphone-8-plus': 'iphone-8-plus',
+   'samsung-galaxy-s8': 'samsung-galaxy-s8',
+   'iphone-6s': 'iphone-6s',
+   'iphone-6': 'iphone-6',
+   'pc-laptop-parts': 'pc-laptop-parts',
+   'iphone-se-2020': 'iphone-se-2020',
+   'samsung-galaxy-s9-plus': 'samsung-galaxy-s9-plus',
+   'apple-mac-mini-parts': 'apple-mac-mini-parts',
+   'ipad-4': 'ipad-4',
+   'samsung-galaxy-s7': 'samsung-galaxy-s7',
+   'ipad-mini-1': 'ipad-mini-1',
+   'iphone-5s': 'iphone-5s',
+   'iphone-6s-plus': 'iphone-6s-plus',
+   'samsung-galaxy-s7-edge': 'samsung-galaxy-s7-edge',
+   'iphone-4': 'iphone-4',
+   'ipad-3': 'ipad-3',
+   'samsung-galaxy-s8-plus': 'samsung-galaxy-s8-plus',
+   'ipad-air-2': 'ipad-air-2',
+   'ipad-6': 'ipad-6',
+   'iphone-6-plus': 'iphone-6-plus',
+   'ipad-air-1': 'ipad-air-1',
+   'ipad-5': 'ipad-5',
+   'iphone-4s': 'iphone-4s',
+   'ipad-2': 'ipad-2',
+   'iphone-5': 'iphone-5',
+   'ipad-mini-retina': 'ipad-mini-retina',
+   'imac-upgrade-kits': 'imac-upgrade-kits',
```

Comment:
Are any of these different? Maybe it'd be clearer to only include the entries where the key is different than the value?

Reply:
Ha! You did this in a later commit!